codegen/module.go

```diff
@@ -673,6 +693,12 @@ type ModuleInstance struct {
 	JSONFileRaw []byte
 }
 
+// GetGeneratedSpec returns the last spec result returned for the module
+// instance
+func (instance *ModuleInstance) GetGeneratedSpec() interface{} {
```

Comment:
nit: per https://golang.org/doc/effective_go.html#Getters, `GenSpec` is the idiomatic Go method name